File: find_cal.py
import re

with open('src/main/resources/static/index.html', 'r', encoding='utf-8') as f:
    html = f.read()

out = []
for i, line in enumerate(html.splitlines()):
    if 'calendar' in line.lower() or 'day-view' in line.lower() or 'time-blocking' in line.lower():
        out.append(f"{i+1}: {line.strip()}")

with open('html_cal.txt', 'w', encoding='utf-8') as f:
    f.write("\n".join(out))

File: find_cal.js
const fs = require('fs');
const text = fs.readFileSync('src/main/resources/static/index.html', 'utf8');

const match = text.match(/function _renderCalendarViews[\s\S]{0,1000}/);
if (match) {
    console.log(match[0]);
} else {
    console.log('Not found _renderCalendarViews');
}

const match2 = text.match(/function renderWeekView[\s\S]{0,1000}/);
if (match2) {
    console.log("\n-----\n" + match2[0]);
}
